sync/p2p_messages.md

```diff
@@ -123,6 +127,13 @@ Message is RLP encoded, fields:
 Each header hash contains a 64-bit big endian height and the corresponding
 hash, see `aec_peer_messages` for details.
 
+## MSG_GET_BLOCK_TXS
+*(Tag = 7)*
```

Comment:
This is correct:
https://github.com/aeternity/epoch/blob/c247b32db2116aa9db700da01aeaae1c2944641e/apps/aecore/include/aec_peer_messages.hrl#L9